src/expressions/closure-expr.md

```diff
@@ -31,8 +31,8 @@ A closure can be forced to capture its environment by copying or moving values b
 This is often used to ensure that the closure's type is `'static`.
 
 The compiler will determine which of the [closure traits](../types/closure.md#call-traits-and-coercions) the closure's type will implement by how it acts on its captured variables.
-The closure will also implement [`Send`](../special-types-and-traits.md#send) and/or [`Sync`](../special-types-and-traits.md#sync) if all of its captured types do.
-These traits allow functions to accept closures using generics, even though the exact types can't be named.
+The closure will also implement the closure traits [`Send`](../special-types-and-traits.md#send) and/or [`Sync`](../special-types-and-traits.md#sync) if all of its captured types do.
+Closure traits allow functions to accept closures using generics, even though the exact types can't be named.
```

Comment:
I wouldn't add "the closure traits" for Send/Sync.  "Closure traits" specifically refers to `Fn`, `FnMut`, and `FnOnce`.  The other traits that closures may implement (send/sync/copy/clone) are just extras, and are not directly related to allowing use as function parameters.

I'm not sure why they are even called closure traits, since they are just the general call traits which are used for things other than closures. And I also question why it explicitly says "using generics", since closures can also be passed as trait objects.

Since the closure type chapter uses a clearer term ("call traits"), I would change it to use that phrasing, and change the last sentence to something like "The call traits allow functions to accept closures as parameters, even though…".